src/servases/XfilesApi.jsx

```diff
@@ -0,0 +1,20 @@
+const getXFilesMain = () => {
+  return fetch('https://xfiles-api.herokuapp.com/api/v1/characters?category=Main_characters', {
```

Comment:
I got cors errors on the fetch. You can fix it with cors anywhere...
  return fetch(`https://cors-anywhere.herokuapp.com/https://xfiles-api.herokuapp.com/api/v1/characters?perPage=20&page=${page}`, {
    headers: {
      Origin: null
    }
  })